src/components/ProjectCard.tsx

```diff
@@ -60,17 +60,17 @@ const ProjectCard: FC<Props> = ({ data }) => {
       </div>
 
       <div className="card-buttons">
-        <a 
-          href={data.githubUrl} 
-          className="button-link" 
-          target="_blank"
+        <a
+          href={data.github_url}
+          className="button-link"
+          target={data.github_url}
```

Comment:
Will this open in the same tab? Same tab vs. new tab?

Reply:
It will open in a new tab

Reply:
👍